Typing error on Eliom >= 8.9.0

Hi,

I can't compile a [test program with a simple server-generated button](https://github.com/ghuysmans/eliom-test/blob/simple_client/test.eliom). I've reduced it to a [simpler program](https://github.com/ghuysmans/eliom-test/blob/simple_client/mini.eliom) and tested different versions of Eliom, first locally then [on GitHub Actions](https://github.com/ghuysmans/eliom-test/runs/4419353689?check_suite_focus=true).

Starting from Eliom 8.9.0, it complains about a `'a Eliom_client_value.t option`:
```
$ eliomc -ppx -infer mini.eliom
File "mini.eliom", line 7, characters 25-46:
7 |     button ~a:[a_onclick [%client fun _ -> ()]] [txt "Click me!"];
                             ^^^^^^^^^^^^^^^^^^^^^
Error: This expression has type 'a Eliom_client_value.t option
       but an expression was expected of type unit -> 'b
       Hint: Did you forget to wrap the expression using `fun () ->'?
Error: Process completed with exit code 2.
```

Have I done something wrong?

Thanks!

--- a/opam
+++ b/opam
@@ -1,6 +1,6 @@
 opam-version: "2.0"
 name: "eliom"
-version: "9.2.1"
+version: "9.2.2"
 maintainer: "dev@ocsigen.org"
 authors: "dev@ocsigen.org"
 synopsis: "Client/server Web framework"

--- a/src/ppx/ppx_eliom_type.ml
+++ b/src/ppx/ppx_eliom_type.ml
@@ -112,7 +112,7 @@ module Pass = struct
       [%e frag_eid] :=
         Some ( Eliom_syntax.client_value "" 0 :
                  [%t typ] Eliom_client_value.t);
-      (Option.get (! [%e frag_eid]) : _ Eliom_client_value.t)
+      (Stdlib.Option.get (! [%e frag_eid]) : _ Eliom_client_value.t)
     ]
 
   let escape_inject

